src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
Avoid magic numbers in your code. If this number is not changable, define a constant at the top of the class with a meaningful varaible name. ex. BALL_NUMEBR_LIMIT
```suggestion
        ball.setBallNumber(new Random().nextInt(BALL_NUMEBR_LIMIT));
```